### PR TITLE
WIP: Fixes GraphQL Upgrader: description inside block 

### PR DESCRIPTION
**Pull request by esjee**
Depends on https://github.com/rmosolgo/graphql-ruby/pull/1151! Let's merge that one first.

Add the scenario for having a description inside the block when defining a field.

### CODE REVIEW

File: spec/graphql/upgrader/member_spec.rb [hunk 1]
```diff
@@ -213,10 +213,54 @@ class UserInterface < BaseInterface
...
+  end
+
+  # describe 'field with description inside block' do
+  #   it 'moves the description as an argument' do
+  #     old = %{
+  #       field :_id, !types.ID, property: :id do
+  #         description 'The primary key from the database'
+  #       end
+  #     }
+  #     new = %{
```
Comment (esjee) [inline spec/graphql/upgrader/member_spec.rb]: I'm unsure if this "new" is as expected!

**Comment by Willianvdv** [comment]
@esjee should we wait for the outcome of the discussion on #1158 before fixing this?

**Comment by rmosolgo** [comment]
Is this change still required? If so, it'll probably need some rework since I change the Upgrader quite a bit 😬

**Comment by rmosolgo** [comment]
Feel free to open a new PR against 1.8-dev if there's still something missing there!

**Pull request closed without merging**